cypress/integration/common/i_click.js

```diff
@@ -20,6 +20,21 @@ Then(`I click third occurrence of {string}`, (clickable) => {
     cy.OPGCheckA11y();
 })
 
+Then(`I click fourth occurrence of {string}`, (clickable) => {
```

Comment:
Could you reduce the repeated code here by creating a "I click {nth} occurrence of {string}"?

Reply:
Started out trying to make it nice English. Then there turned out to be lots of occurrences.! Have now changed this to "I click occurrence {int} of {string}" . Note that this counts up from 0. Toyed with the idea of doing -1 and having it start at 1 in the feature file, but thought that would be confusing